utils/resource-monitor/graph_pidstat.py

```diff
@@ -0,0 +1,215 @@
+'''
+Graphs the performance of the batch job with HAPI as the source 
+
+This module runs a batch job with HAPI as the source and produces a graph and csv files of the performance of 
+the HAPI server, postgres database and the pipeline.
+
+Example usage: 
+    python3 graph_pidstat.py \
+      --numProc 12  --dataDescription small_dataset \
+      --outputParquetPath /tmp/hapi-test/  --outputResultsPath /tmp/hapi-performance/
+'''
+
+import sys
+import os
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
Please avoid `import ... as ...` unless there are conflicts or the short name is "standard"; [style guide](https://google.github.io/styleguide/pyguide.html#22-imports)